HOWTO.adoc

```diff
@@ -35,18 +36,38 @@ local emin, emax = vm:read_from_map(p1, p2) -- Emerged minimum and maximum posit
 
 === Grammar and Punctuation
 * Always capitalize proper names and first letters in plain text.
- ** Capitalization in quotation marks is not required and usually discouraged. The goal of quotation in documentation is to mirror _exactly_.
+** Capitalization in quotation marks is not required and usually discouraged. The goal of quotation in documentation is to mirror _exactly_.
 * End all plain text with a period. This does not apply to code examples.
 * Use the Oxford (or serial) comma. It helps reduce confusion.
 
+=== Admonitions
+The use of Admonitions (`NOTE:`, `TIP:`, `IMPORTANT:`, `CAUTION:`, and `WARNING:`) are useful for bringing the reader's attention to a specific idea or statement.
```

Comment:
I'd remove the colons after each admonition, it looks weird with them

Reply:
I wanted it to be the syntax (it is monospaced, after all) but it certainly doesn't need to be.